prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Creation</title>
+</head>
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    var $container = document.querySelector('#container');
+    console.log('This variable contains:', $container);
+
+    var $heading = document.createElement('h1');
+    console.log($heading);
+
+    document.getElementById('container').textContent = 'Hello World!';
```

Comment:
Try adding your text content to the heading instead of the container.

```js
$heading.textContent = 'Hello World!';
```

Reply:
<img width="1440" alt="Screen Shot 2022-06-18 at 10 09 19 AM" src="https://user-images.githubusercontent.com/106715675/174449270-4b4c3c80-4c00-4d5f-a47a-9c0c92ca6dcb.png">